imagetest/test_suites/hotattach/hot_attach_test.go

```diff
@@ -173,10 +173,7 @@ func TestFileHotAttach(t *testing.T) {
 	}
 	// mount again, then read from the file
 	if runtime.GOOS == "windows" {
-		procStatus, err := utils.RunPowershellCmd("Initialize-Disk -PartitionStyle GPT -Number 1 -PassThru | New-Partition -DriveLetter " + windowsMountDriveLetter + " -UseMaximumSize | Format-Volume -FileSystem NTFS -NewFileSystemLabel 'Perf-Test' -Confirm:$false")
-		if err != nil {
-			t.Fatalf("failed to initialize disk on reattach for windows: errors %v %s %s", err, procStatus.Stdout, procStatus.Stderr)
-		}
+		t.Log("windows disk was successfully reattached")
```

Comment:
Is this just skipping the Windows test for now?

Reply:
It's now passing the windows test. It turns out that after first initializing the disk, then detaching and reattaching, the powershell command to initialize the disk again is not necessary because it remembers that the disk was initialized.

This test does verify that the text file can still be read after hot attach.